aci/resource_aci_cloudsvcepg.go

```diff
@@ -0,0 +1,1155 @@
+		DeleteContext: resourceAciCloudServiceEPgDelete,
+
+		Importer: &schema.ResourceImporter{
+			State: resourceAciCloudServiceEPgImport,
+		},
+
+		SchemaVersion: 1,
+		Schema: AppendBaseAttrSchema(AppendNameAliasAttrSchema(map[string]*schema.Schema{
+			"cloud_applicationcontainer_dn": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"access_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"Private",
+					"Public",
+					"PublicAndPrivate",
+					"Unknown",
+				}, false),
+			},
+			"azure_private_endpoint": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"custom_service_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+			},
+			"deployment_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"CloudNative",
+					"CloudNativeManaged",
+					"Third-party",
+					"Third-partyManaged",
+					"Unknown",
+				}, false),
+			},
+			"flood_on_encap": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"disabled",
+					"enabled",
+				}, false),
+			},
+			"label_match_criteria": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"All",
+					"AtleastOne",
+					"AtmostOne",
+					"None",
+				}, false),
+			},
+			"name": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"preferred_group_member": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"exclude",
+					"include",
+				}, false),
+			},
+			"prio": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"level1",
+					"level2",
+					"level3",
+					"level4",
+					"level5",
+					"level6",
+					"unspecified",
+				}, false),
+			},
+			"cloud_service_epg_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ValidateFunc: validation.StringInSlice([]string{
+					"Azure-ADDS",
+					"Azure-AksCluster",
+					"Azure-ApiManagement",
+					"Azure-ContainerRegistry",
+					"Azure-CosmosDB",
+					"Azure-Databricks",
+					"Azure-KeyVault",
+					"Azure-Redis",
+					"Azure-SqlServer",
+					"Azure-Storage",
+					"Azure-StorageBlob",
+					"Azure-StorageFile",
+					"Azure-StorageQueue",
+					"Azure-StorageTable",
+					"Custom",
+					"Unknown",
+				}, false),
+			},
+
+			"relation_cloud_rs_cloud_epg_ctx": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "Create relation to fv:Ctx",
+			},
+			"relation_fv_rs_cons": {
```

Comment:
I think the sub-attributes of the attribute having TypeSet needs to be Computed. Let me get back to you on this.